src/main/java/org/mitre/synthea/world/agents/Person.java

```diff
@@ -89,10 +89,18 @@
   public final JDKRandomGenerator random;
   public final long seed;
   public long populationSeed;
+  /** 
+   * Tracks the last time that the person was updated over a serialize/deserialize.
+   */
+  public long lastUpdated;
+  /**
+   * Tracks the remaining modules for a person over a serialize/deserialize.
+   */
+  public List<Module> currentModules;
   public Map<String, Object> attributes;
   public Map<VitalSign, ValueGenerator> vitalSigns;
-  private Map<String, Map<String, Integer>> symptoms;
-  private Map<String, Map<String, Boolean>> symptomStatuses;
+  Map<String, Map<String, Integer>> symptoms;
+  Map<String, Map<String, Boolean>> symptomStatuses;
```

Comment:
Is there are reason to change these to default visibility?

Reply:
Just for testing purposes. See PersonTest#testSerializationAndDeserialization.